massa-bootstrap/src/tools.rs

```diff
@@ -5,11 +5,15 @@ use std::net::IpAddr;
 /// it would fail the comparison with a canonicalized ipv4 from the config
 /// (eg. Ipv4 is not converted to ipv6 by canonicalize)
 pub(crate) fn normalize_ip(ip: IpAddr) -> IpAddr {
-    match ip {
+    let ip = match ip {
```

Comment:
this "match" part can be removed and only have the implem as in https://doc.rust-lang.org/src/core/net/ip_addr.rs.html#1733

Reply:
So the comment just above is wrong ?

Reply:
Removed. Just wondering why it was there before with a big comment